### PR TITLE
Update fOUTje board pic

### PR DESCRIPTION
**Pull request by lserranoruber**

### CODE REVIEW

**Review by laurensWe — APPROVED** [review]
LGTM

File: .gitignore
Comment (thomcsmits) [inline .gitignore:1]: Maybe nicer to just create a .env rather than adding all possible virtual environment options here :p 

Rest looks good

Comment (lserranoruber) [reply .gitignore:1]: fineee im just bad with environments lol

**Review by thomcsmits — APPROVED** [review]
LGTM!

**Pull request merged**